src/DynamoCore/Models/NodeLoaders/ZeroTouchNodeLoader.cs

```diff
@@ -22,7 +23,7 @@ public ZeroTouchNodeLoader(LibraryServices libraryServices)
             this.libraryServices = libraryServices;
         }
 
-        public NodeModel CreateNodeFromXml(XmlElement nodeElement, SaveContext context)
+        public NodeModel CreateNodeFromXml(XmlElement nodeElement, SaveContext context, ElementResolver resolver)
```

Comment:
Why do zero touch nodes need an `ElementResolver`? It should be applicable only to CBN's and input nodes, right?